lib/archive/diff_test.go

```diff
@@ -264,6 +264,8 @@ func TestDiffNoData(t *testing.T) {
 }
 
 func TestDiffFilterSpec(t *testing.T) {
+	t.Skip()
```

Comment:
Pretty sure I saw somewhere why we are skipping all of these now, but I can't remember now.  Can you put the reason into the body of the PR and update the body for that matter too, as I don't think that the original comment is still valid?